packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,762 @@
+            <li class="qs-step__error-list-item">Make necessary adjustments</li>
+            <li class="qs-step__error-list-item">
+              <a href="#">Try again to install</a>
+            </li>
+          </ol>
+          <a
+            class="qs-step__error-button qs-button qs-button--bordered"
+            href="https://appland.com/company/contact-us"
+            target="_blank"
+            rel="noopener noreferrer"
+            >For further assistance please contact us</a
+          >
+        </div>
+        <div class="qs-step__success" v-if="step1Completed && !error">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            Agent installed
+          </span>
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : Configure AppMap ->
+          </button>
+        </div>
+      </section>
+      <section class="qs-step" v-if="currentStep === 2">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Configure AppMap</h1>
+        </div>
+        <div class="qs-step__block" v-if="!step2Completed">
+          <p>
+            This will create an <a href="#">appmap.yml</a> config file in the
+            root directory of your project. This will tell AppMap what code to
+            record. You can run these defaults or add more packages, gems, or
+            specific functions. You can edit this file at any time.
+          </p>
+          <code class="qs-code" @click="select">{{ appmapYmlSnippet }}</code>
+          <button class="qs-button" v-if="!isActionRunning" @click="runAction">
+            Create appmap.yml config file
+          </button>
+          <div class="qs-loader" v-if="isActionRunning">
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__success" v-if="step2Completed">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            AppMap configuration file created
+          </span>
+          <a
+            href="#"
+            class="qs-step__success-link"
+            @click.prevent="viewAppmapYml"
+            >appmap.yml</a
+          >
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : Record AppMaps ->
+          </button>
+        </div>
+      </section>
+      <section class="qs-step" v-if="currentStep === 3">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Record AppMaps</h1>
+          <select class="qs-select">
+            <option>minitest (detected)</option>
+          </select>
```

Comment:
The options here will also need to be provided via prop. All options will technically be detected, so we won't need that status explicitly called out.

We'll also need a way of knowing what option the user has selected, maybe it's provided via another param on `onAction`?

```ts
onAction(language: string, step: number, testFramework?: string | null | undefined): Promise<void>
```